EpicCactus94.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.
+
+## Plan
+
+I've already begun designing the .stl files which will be used to print the cube, and I've assured access to 3D printers before the building week. 
+I'll buy some screws and bolts to act as weights out of my own pocket as they're pretty cheap. Once the parts come in I should be ready to begin assembly 
+day 1, and focus more on programming my project the rest of the week or so.

```

Comment:
What does programming involve? What languages or frameworks are you using? Where is the code going to run?

Reply:
All of the computation will be done through an Arduino microcontroller. Once all of the electrical parts of the cube (motors, gyroscope, battery, etc.) are connected to the Arduino, I will develop a program and upload it to the Arduino. It will be programmed in the Arduino IDE which uses C++, and there are specific libraries like the DC motors library which will help me actually program it. The general plan for the code is to take in readings from the gyroscope to determine if the cube is about to fall over. Once that's done, the cube should adjust its motors so that it can stay balanced.

Reply:
That sounds like a more compelling explanation of your project. Could you add this to your project proposal?

Reply:
Sure. I added it to the .md file in this pull request